diagnostic.md

```diff
@@ -35,7 +35,13 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    The first actually creates 2 routes -- the path '[domain]/products' renders the 'products' template, and the path '[domain]/products/[product id]' renders the products template, with the product template appearing inside that page (wherever the {{outlet}} tag appears)
+
+    The second simply routes from '[domain]/products/[product id]' to the product template.
+
+    So, first difference: the nested route creates two different url routes,
+
+    and second, the nested route has the product template nested inside the products template, while the single line just renders the product template at the appropriate url
```

Comment:
These would both create the same route in different ways: 
The first `'product'` route definition is nested under the `'products'`
definition, whereas the second `'product'` route definition is a sibling of
the `'products'` definition.

Nested route definitions, like the first `'product'` route above, require
a nested file structure, such that Route and Template files for `'product'`
are expected to be found at `app/products/product`. For sibling routes,
folders are siblings as well.

The first `'product'` route definition instructs Ember to render
`app/products/product/template.hbs` in the context (the `outlet`) of the
parent `app/products/template.hbs`. The second `'product'` route definition
instructs Ember to render `app/product/template.hbs` in the context of the
application template.

These two route definitions do **not** have different paths.

0/1